honeybee_energy_standards/lib/schedules.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+"""Extend the honeybee_energy schedule library."""
+from honeybee_energy.schedule.ruleset import ScheduleRuleset
+
+from honeybee_energy.lib.schedules import _idf_schedules
+
+import os
+import json
+
+
+# load the standards gem data of schedules to Python dictionaries.
+_data_dir = os.path.join(os.path.dirname(__file__), '../data')
+
+_schedule_dir = os.path.join(_data_dir, 'schedule.json')
+with open(_schedule_dir, 'r') as f:
+    _schedule_standards_dict = json.load(f)
```

Comment:
What if we just add all the objects to _idf_schedules on load and then look up into it directly when we need it? That will save us from loading them every time and it doesn't look like it will take too much RAM. As I brought it up before for all these cases it looks like using a database will make the implementation cleaner. If we haven't I think we should open an issue and take a look at implementing it once we passed the current deadlines.

Reply:
If we initialize all of the Python objects when we load all of the data from the gem, it ends up taking a lot longer to run the first time because of all the checks that happen when you make a new Python object. And I don't follow how sending everything to a Python object instead of storing it in its loaded dictionary form relates to the SQL database. Isn't necessitating that everything be a Python object from the get-go putting us further from the idea of having everything in a database and only loading the Python object when we need to.

Reply:
The comment about database is about the overall design / approach on how we are storing and loading the standard library not this specific instance. We will get there when we get there.